docs/reference/env-variables.rst

```diff
@@ -0,0 +1,943 @@
+        | Sets the maximum size (in KB) for sub-allocations.
+      - ``4096``
+      -
+
+    * - | ``GPU_MAX_USWC_ALLOC_SIZE``
+        | Sets  the maximum USWC allocation size in MB.
+      - ``2048``
+      - -1: No limit
+
+    * - | ``GPU_MAX_WORKGROUP_SIZE``
+        | Sets the maximum number of workitems in a workgroup for GPU.
+      - ``0``
+      - 0: Ignore the environment variable and use the default workgroup size, which is 256.
+
+    * - | ``GPU_MIPMAP``
+        | Enables GPU mipmap extension.
+      - ``1``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_NUM_COMPUTE_RINGS``
+        | Sets the number of GPU compute rings.
+      - ``2``
+      - | 0: Disable
+        | 1,2, etc. ...: Number of compute rings
+
+    * - | ``GPU_NUM_MEM_DEPENDENCY``
+        | Sets the number of memory objects for dependency tracking.
+      - ``256``
+      -
+
+    * - | ``GPU_PINNED_MIN_XFER_SIZE``
+        | Sets the minimum buffer size (in MB) for pinned read and write transfers.
+      - ``128``
+      -
+
+    * - | ``GPU_PINNED_XFER_SIZE``
+        | Sets the buffer size (in MB) for pinning in read and write transfers.
+      - ``32``
+      -
+
+    * - | ``GPU_PRINT_CHILD_KERNEL``
+        | Prints the specified number of child kernels.
+      - ``0``
+      -
+
+    * - | ``GPU_RESOURCE_CACHE_SIZE``
+        | Sets the resource cache size in MB.
+      - ``64``
+      -
+
+    * - | ``GPU_SINGLE_ALLOC_PERCENT``
+        | Maximum size of a single allocation as percentage of total.
+      - ``85``
+      - 
+
+    * - | ``GPU_STAGING_BUFFER_SIZE``
+        | Size of the GPU staging buffer in megabyte (MB).
+      - ``4``
+      -
+
+    * - | ``GPU_STREAMOPS_CP_WAIT``
+        | Force the stream memory operation to wait on CP.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_USE_DEVICE_QUEUE``
+        | Use a dedicated device queue for the actual submissions.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_WAVES_PER_SIMD``
+        | Force the number of waves per SIMD. 1-10
+      - ``0``
+      - 
+
+    * - | ``GPU_XFER_BUFFER_SIZE``
+        | Transfer buffer size for image copy optimization in KB.
+      - ``0``
+      -
+        
+    * - | ``HIP_FORCE_DEV_KERNARG``
+        | Force device memory for kernel args.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``HIP_HIDDEN_FREE_MEM``
+        | Amount of memory to hide from the free memory reported by hipMemGetInfo.
+      - ``0``
+      - 0: Disable
+
+    * - | ``HIP_HOST_COHERENT``
+        | Coherent memory in ``hipHostMalloc``.
+      - ``0``
+      - | 0: Memory is not coherent between host and GPU.
+        | 1: Memory is coherent with host.
+        | Environment variable has effect, if:
+        | - One of the HostMalloc flags is set.
+        | - ``hipHostMallocCoherent=0``
+        | - ``hipHostMallocNonCoherent=0``
+        | - ``hipHostMallocMapped=0``
```

Comment:
```suggestion
      - | 0: Memory is not coherent.
        | 1: Memory is coherent.
        | Environment variable has effect, if one of the following ``HostMalloc`` flags is set.
        | - ``hipHostMallocCoherent=1``
        | - ``hipHostMallocNonCoherent=1``
        | - ``hipHostMallocMapped=1``
```
To set flag such as hipHostMallocCoherent and the other two, don't we need a value of 1? The value 0 doesn't look right to me.

Reply:
Please check my fix.